tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+  "defaultSeverity": "error",
+  "extends": [
+    "tslint-config-airbnb"
+  ],
+  "rules": {
+    "trailing-comma": [true]
+  }
+}
```

Comment:
EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon:
![image](https://user-images.githubusercontent.com/10209180/126420285-15f4d88e-9a11-40c2-8eda-d3d4634cb332.png)